Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1824,11 +1826,28 @@ def update_input(self,vtkobjects,array1,array2=None,update=True):
                       mapper = missingMapper2
                       wrp = a[2]
                   else:
-                      mapper = vtk.vtkPolyDataMapper()
-                      mapper.SetInputConnection(ports[i].GetOutputPort())
-                      lut,rg = vtkobjects["vtk_backend_luts"][i]
-                      if lut is not None:
-                          mapper.SetLookupTable(lut)
+                      ## Labeled contours are a different kind
+                      if vtkobjects.has_key("vtk_backend_luts"):
+                          lut,rg = vtkobjects["vtk_backend_luts"][i]
+                          mapper = vtk.vtkPolyDataMapper()
+                      elif vtkobjects.has_key("vtk_backend_labeled_luts"):
+                          lut,rg = vtkobjects["vtk_backend_labeled_luts"][i]
+                          mapper = vtk.vtkLabeledContourMapper()
+                      if lut is None:
+                          mapper.SetInputConnection(ports[i].GetOutputPort())
+                      else:
+                          if mapper.IsA("vtkPolyDataMapper"):
+                              mapper.SetInputConnection(ports[i].GetOutputPort())
+                              mapper.SetLookupTable(lut)
+                          else:
+                              stripper = vtk.vtkStripper()
+                              stripper.SetInputConnection(ports[i].GetOutputPort())
+                              mapper.SetInputConnection(stripper.GetOutputPort())
+                              stripper.Update()
+                              tprops = vtkobjects["vtk_backend_contours_labels_text_properties"]
+                              mapper.GetPolyDataMapper().SetLookupTable(lut)
+                              mapper.SetLabelVisibility(1)
+                              mapper.SetTextProperty(tprops)
```

Comment:
What we've done in some of the ParaView tests is to run an animation from start to finish and check certain key frames (usually just the first and the last). It seems to work rather well at catching errors.

VCS should be able to do something similar, AIUI. We'd just need to call the image comparison code at multiple points in the test.